messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/MessagingSubsystemParser_1_0.java

```diff
@@ -261,41 +261,43 @@
                                                 .addChild(
                                                         builder(SecurityRoleDefinition.INSTANCE)
                                                                 .addAttributes(
-                                                                        SecurityRoleDefinition.SEND,
-                                                                        SecurityRoleDefinition.CONSUME,
-                                                                        SecurityRoleDefinition.CREATE_DURABLE_QUEUE,
-                                                                        SecurityRoleDefinition.DELETE_DURABLE_QUEUE,
-                                                                        SecurityRoleDefinition.CREATE_NON_DURABLE_QUEUE,
+                                                                        SecurityRoleDefinition.MANAGE,
                                                                         SecurityRoleDefinition.DELETE_NON_DURABLE_QUEUE,
-                                                                        SecurityRoleDefinition.MANAGE)))
+                                                                        SecurityRoleDefinition.CREATE_NON_DURABLE_QUEUE,
+                                                                        SecurityRoleDefinition.DELETE_DURABLE_QUEUE,
+                                                                        SecurityRoleDefinition.CREATE_DURABLE_QUEUE,
+                                                                        SecurityRoleDefinition.CONSUME,
+                                                                        SecurityRoleDefinition.SEND
```

Comment:
WFCORE-1644 is now fixed in full. I'll have a look and see what's still broken re: unnecessary differences after marshalling.